README.md

```diff
@@ -4,6 +4,8 @@
 
 This package contains a plugin that allows you to natively lint JSON and JSONC files using ESLint.
 
+**Important:** This plugin requires ESLint v9.6.0 or higher and you must be using the [new configuration system](https://eslint.org/docs/latest/use/configure/configuration-files).
```

Comment:
Maybe we should specify this in package.json `peerDependencies`?

Reply:
Ah good idea!